src/components/campaign/campaign.test.ts

```diff
@@ -50,36 +55,51 @@ describe("Find on shelf modal - default", () => {
     );
   });
 
-  it("Shows a full campaign", () => {
-    // Intercept campaign data call - full campaign.
+  it("Shows a full campaign with image, text & link", () => {
     cy.fixture("search-result/campaign.json")
       .then((result) => {
-        cy.intercept("POST", "**/dpl_campaign/match", result);
+        cy.intercept("**/dpl_campaign/match", result);
       })
-      .as("Campaign service");
+      .as("Campaign service - full campaign");
 
-    cy.get("section").contains("Lorem ipsum Harry Potter");
+    cy.get("img")
+      .should("be.visible")
+      .and("have.prop", "naturalWidth")
+      .should("be.greaterThan", 0);
```

Comment:
I made it into a helper function called isImageLoaded(), let's see if it can be reused other places though. Because it just takes the first image. Maybe it could be refactored in the future depending on the use cases.